### PR TITLE
Adding code to highlight changes done for radio selection.

### PR DESCRIPTION
**Pull request by parvinba**
https://collaboration.homeoffice.gov.uk/jira/browse/ASL-4533

### CODE REVIEW

File: client/components/diff-window.js [hunk 1]
```diff
@@ -304,6 +320,12 @@ const DiffWindow = (props) => {
   const compare = () => {
 
     const hasVisibleChanges = hasContentChanges(before, props.value, props.type);
+    let removed = changes.removed;
+    let added = changes.added;
```
Comment (kiran-varma-home-office) [inline client/components/diff-window.js:324]:
```suggestion
    let { removed, added } = changes;
```

File: client/components/diff-window.js [hunk 2]
```diff
@@ -304,6 +320,12 @@ const DiffWindow = (props) => {
...
+    if (props.type === 'radio') {
+      removed = { added: false };
+      added = { added: true };
+    }
```
Comment (kiran-varma-home-office) [inline client/components/diff-window.js:328]: compare function has only render at the moment, I am thinking maybe its good to move above to renderDiff !

also, I don't see `removed = { added: false };` used anywhere, am I missing something !?

Comment (parvinba) [reply client/components/diff-window.js:328]: It is too late for renderdiff to determine if it is added or removed. Existing comparison for checkbox does it here. Removed is passed in previous window (line 347)

File: client/components/diff-window.js (hunk 1 above)
Comment (parvinba) [reply client/components/diff-window.js:324]: done..

File: client/components/diff-window.js (hunk 2 above)
Comment (kiran-varma-home-office) [reply client/components/diff-window.js:327]: just a thought, i think you could remove all this and do everything in radioDiff function you created .. would this work? ( I didn't test it, I just typed quickly)

`        const radioDiff = (parts) => {
      const newParts = { ...parts, removed: { added: false }, added: { added: true }}
      return (
        <p>
          {
            value === undefined ? (
              <em>{ DEFAULT_LABEL }</em>
            ) : (
              <span className={`diff ${newParts.added ? 'added' : 'removed'}`}> { value ? 'Yes' : 'No'}</span>
            )
          }
        </p>
      );
    };`

Comment (parvinba) [reply client/components/diff-window.js:327]: No it will not work..

**Review by kiran-varma-home-office — APPROVED** [review]
I am happy to approve but please get other opinions 👍

**Review by jeff-horton-ho-sas — APPROVED** [review]

**Pull request merged**